### PR TITLE
[Execute] Revisit executeQuickInput file

### PR DESCRIPTION
**Pull request by YongseopKim**
Let's revisit executeQuickInput file.

ONE-vscode-DCO-1.0-Signed-off-by: Yongseop Kim <yons.kim@samsung.com>

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -283,7 +283,7 @@
       "commandPalette": [
         {
           "command": "one.backend.infer",
-          "when": "false"
+          "when": "true"
```
Comment (YongseopKim) [inline package.json:286]: After checking some tests, this will be reverted.

**Comment by YongseopKim** [comment]
PTAL @yunjayh and @Samsung/one-vscode

File: package.json (hunk 1 above)
Comment (yunjayh) [reply package.json:286]: Q) Will this be going to be merged and reverted after some tests in main branch?

**Review by yunjayh — DISMISSED** [review]
I carefully read this pr, and it looks so great to me. :+1: Thank you for your effort!

File: package.json (hunk 1 above)
Comment (YongseopKim) [reply package.json:286]: No. If you agree on this design, I'll add new patch to revert this on this PR.

Comment (YongseopKim) [reply package.json]: Done

**Comment by YongseopKim** [comment]
@jyoungyun I have a plan to revisit View/InstallQuickInput like this way. Any plan to add new features for InstallQuickInput?

**Review by yunjayh — APPROVED** [review]

**Comment by YongseopKim** [comment]
@jyoungyun this is fixing core code. I don't touch any ui flow. draft: https://github.com/Samsung/ONE-vscode/pull/990

**Comment by YongseopKim** [comment]
PTAL @llFreetimell

**Comment by jyoungyun** [comment]
> this is fixing core code. I don't touch any ui flow. draft: https://github.com/Samsung/ONE-vscode/pull/990

@YongseopKim I don't know the existing ui flow about this changes, so I need some information to understand your changes.
I have several comments about your changes. I will update the comment on #990 PR.

**Review by llFreetimell — APPROVED** [review]

**Comment by jyoungyun** [comment]
I haven't tested this code. By the way, does this commit go back to the previous step?

**Comment by llFreetimell** [comment]
I will not merge until discussion between @jyoungyun and @YongseopKim is ended :)

**Comment by jyoungyun** [comment]
> I haven't tested this code. By the way, does this commit go back to the previous step?

I was confirmed by @yunjayh that this feature did not originally support go back to the previous step. So ignore my comment.

**Pull request merged**